src/track/bpm.h

```diff
@@ -25,6 +25,12 @@ class Bpm final {
 
     // Adjusts floating-point values to match their string representation
     // in file tags to account for rounding errors.
+    // NOTE(2019-02-19, uklotzde): The pre-normalization cannot prevent
```

Comment:
Is this a to-do?

Reply:
No, this is a fact resulting from the ID3v2 format.